client/src/AppBundle/Service/DocumentSyncService.php

```diff
@@ -24,87 +29,223 @@ class DocumentSyncService
     /** @var S3Storage */
     private $storage;
 
-    public function __construct(S3Storage $storage)
+    /**
+     * @var SiriusApiGatewayClient
+     */
+    private $siriusApiGateWayClient;
```

Comment:
A minor thing, but the casing of this isn't quite right
```suggestion
    private $siriusApiGatewayClient;
```

Reply:
Needs updating a few other places in that file, sorry didn't clock that 🤦‍♂

Reply:
My bad! Updated now.